### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by smari** [mailpile/Mailpile#448]
I just did some rough accessibility testing with a user with impaired vision who uses a high contrast display and a screen reader. The results were surprisingly good, according to the user, who said that GMail is completely unusable in the normal mode, so they have a special accessible mode. Given that Mailpile has not been developed with particular regard to accessibility so far, it's impressively functional to blind/vision impaired users. 

There are a few items that could be made better though:
- The top navigation buttons have no alt= text, so tabbing through them doesn't give any context for the screen reader. (Hovering over them does give good descriptions though)
- The search box should select/highlight the current search term when it is selected, otherwise a user will just start typing without seeing that there is preexisting text. So instead of typing "banana" as expected, or "banana in:inbox" which would have worked, the result was "bananain:inbox", which is meaningless.
- Save and send buttons in compose need alt text. If the send button were made into a submit, it's importance to the form would be evident to the screen reader. Currently nothing in them gets read (even an a-tag would be better)
- When tabbing to a to:/cc:/bcc: box that has an e-mail in it which has an associated fingerprint, the screen reader starts reading the entire fingerprint.
- Edit boxes need alt-texts. It is currently hard to determine whether the cursor is in the subject, to, cc, bcc or main text box, or somewhere else entirely.
- When tabbing through the left-side tag links, the screen reader says "Inbox 140", which would be better if read as "Inbox 140 unread". If we can make screen readers pick up "unread" without showing that otherwise, it would be easier to use
- The search box needs an alt tag.
